docs/rest.md

```diff
@@ -240,6 +245,8 @@ Sample result:
     * `update_available`: if present and not empty, it means the snap can be
       updated to the version specified as a value to this entry.
     * `channel`: which channel the package is currently tracking.
+    * `prices`: if present the snap needs to be purchased before installing. It's
+      a dict with the available currency codes as keys and the prices as values.
```

Comment:
it's not a dict, because json isn't python :-) it's an object which member names are the ISO 4217 currency codes etc etc.